deployment/ansible/group_vars/all.example

```diff
@@ -79,3 +79,14 @@ keystore_password: 'driverkeystore'
 
 ## Django allowed hosts setting. Override when not in DEBUG mode.
 allowed_host: ''
+
+## Access settings for Monit admin interface. Note that this is
+## insecure; ideally, SSL should be used (which Monit supports).
+## An intermediate step would be to restrict the IPs from which
+## monit can be accessed via monit_allow_hosts
+## Username and Password MUST be defined in order for monit to work
+## but monit is restricted to requests coming from localhost by default
+monit_allow_hosts: []
```

Comment:
Cool, I like how this is made configurable. Setting it to the couple IPs of the small number of people who will potentially access monit will be a pretty good interim solution to start with.